sky/serve/constants.py

```diff
@@ -69,7 +69,7 @@
 # automatically generated from this start port.
 CONTROLLER_PORT_START = 20001
 LOAD_BALANCER_PORT_START = 30001
-LOAD_BALANCER_PORT_RANGE = '30001-30100'
+LOAD_BALANCER_PORT_RANGE = '30001-30020'
```

Comment:
Do you mean revert back to the original `'30001-30100'`? That still causes issues if the user is using `LoadBalancer` mode since it will create many services to expose the large number of ports.